src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -61,10 +63,33 @@ function createMutationObserverCallback() {
   };
 }
 
-function renderHeader(megaMenuData, headerContainer) {
+function renderHeader(megaMenuData = [], headerContainer) {
+  // HARDCODED: Since feature toggles aren't available in this context,
+  // we're using a hardcoded value. Set to true or false based on desired behavior.
+  const showMhvHeaderLinks = false;
```

Comment:
I'm hardcoding a "feature toggle" here because it doesn't seem like we can access the feature toggles in this context, or, it's taking too long for them to load. There are no lower environments to test a Teamsite site anyway so we pretty much only have local testing available.